tools/wpt/browser.py

```diff
@@ -1091,28 +1102,31 @@ def webdriver_supports_browser(self, webdriver_binary, browser_binary):
             return True
 
         # Check that the EdgeDriver version matches the Edge version.
-        edgedriver_major = edgedriver_version.split('.')[0]
-        browser_major = browser_version.split('.')[0]
+        edgedriver_major = int(edgedriver_version.split('.')[0])
+        browser_major = int(browser_version.split('.')[0])
         if edgedriver_major != browser_major:
-            self.logger.warning("EdgeDriver %s does not match Edge %s" %
-                                (edgedriver_version, browser_version))
+            self.logger.warning(
+                "EdgeDriver %s does not match Edge %s" %
+                (edgedriver_version, browser_version))
             return False
         return True
 
-    def install_webdriver(self, dest=None, channel=None, browser_binary=None):
-        if self.platform != "win" and self.platform != "macos":
-            raise ValueError("Only Windows and Mac platforms are currently supported")
-
+    def install_webdriver_by_version(self, version, dest=None):
```

Comment:
The `version` argument here appears to go unused, which also makes the name `install_webdriver_by_version` suspect. Does this need renaming to match what it really does, maybe `install_webdriver_for_dev_channel` if that's the intention?

Reply:
Thanks, that was a bug. Updated this function to use version correctly.